jwst/datamodels/slit.py

```diff
@@ -184,20 +205,39 @@ def __init__(self, init=None, data=None, dq=None, err=None,
 
         if dq is not None:
             self.dq = dq
+
         if err is not None:
             self.err = err
+
         if bunit_data is not None:
             self.meta.bunit_data = bunit_data
+
         if bunit_err is not None:
             self.meta.bunit_err = bunit_err
+
         if name is not None:
             self.name = name
 
         if wavelength_pointsource is not None:
             self.wavelength_poointsource = wavelength_pointsource
+
         if pathloss_pointsource is not None:
             self.pathloss_pointsource = pathloss_pointsource
+
         if wavelength_uniformsource is not None:
             self.wavelength_uniformsource = wavelength_uniformsource
+
         if pathloss_uniformsource is not None:
             self.pathloss_uniform_source = pathloss_uniformsource
+
+        if pathloss_pointsource2d is not None:
+            self.pathloss_pointsource2d = pathloss_pointsource2d
+
+        if pathloss_pointsource2d is not None:
```

Comment:
The second pathloss_pointsource2d was probably supposed to be wavelength_pointsource2d, either way, it isn't needed